overlord/snapstate/handlers.go

```diff
@@ -662,23 +661,12 @@ func (m *SnapManager) undoPrepareSnap(t *state.Task, _ *tomb.Tomb) error {
 
 	// Only report and error if there is an actual error in the change,
 	// we could undo things because the user canceled the change.
-	var isErr bool
 	for _, tt := range t.Change().Tasks() {
 		if tt.Status() == state.ErrorStatus {
-			isErr = true
+			tt.Logf("task status failed: %s %s", tt.ID(), tt.Kind())
```

Comment:
I was unsure about this change. If we don't want this log line, then we can remove this entire loop.

Reply:
we don't want this afaict, it's just attaching info again to the task that is already there as the task status is ErrorStatus, no? or am I missing something